server/index.js

```diff
@@ -17,22 +17,41 @@ app.get('/students', function (req, res) {
     })
 })
 
+app.get('/students/:id', (req, res) => {
+  const studentId = parseInt(req.params.id, 10)
```

Comment:
You don't need to `parseInt`. As long as the string is an integer number, `knex` can handle it.